docs/install/Knative-with-MicroK8s.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Install on MicroK8s"
+linkTitle: "MicroK8s"
+weight: 10
+type: "docs"
+---
+
+This guide walks you through the installation of Knative using [MicroK8s](https://microk8s.io).
+
+You can find [guides for other platforms here](./README.md).
+
+### Install MicroK8s
+
+```shell
+   sudo snap install --classic microk8s
+```
+
+## Alias MicroK8s kubectl for convenience
+
+```shell
+   sudo snap alias microk8s.kubectl kubectl
+```
+
+## Enable Knative
+
+```shell
+   echo 'N;' | microk8s.enable knative
+```
```

Comment:
Can you add a description of what gets installed and which version. This feels a bit too magical with little information

Reply:
+1 It would be great to know what version gets installed and how this will be kept up-to-date.

Reply:
@ammarn911  Can you explicitly list what gets installed with this command? With that change, I'll be able to approve and LGTM this so it merges.

Reply:
@samodell done. Added what gets installed; Knative Serving v0.7.1 and Knative Eventing v0.7.1

Reply:
@ammarn911 do you know if these versions are updated and reflect the current versions?
I am working on #1850 so I was wondering if you can confirm whether we can remove mention of the versions or not.
Thanks! :slightly_smiling_face: